src/triggers/AfterInsertExample_Trigger.trigger

```diff
@@ -0,0 +1,13 @@
+trigger AfterInsertExample_Trigger on Account (after insert) {
+    for(Account a : Trigger.New) {
+        String accountName = a.Name;
+        String accountNo = a.AccountNumber;
+        String emailAddress = a.Email__c;
+        EmailManager.sendMail(emailAddress, 'Congratulations for creating acconut', 'Congratulations for creating account your account details are Account Number = ' + accountNo + ', Account Name = ' + accountName);
+        System.debug('Email Send Successfully at: ' + emailAddress);
+    }
+}
```

Comment:
have you executed this, if so how.
we are not allowed to do some events inside of the FOR loop but you are sending email.